tests/distributions/test_continuous.py

```diff
@@ -444,6 +444,19 @@ def test_exponential(self):
             lambda q, lam: st.expon.ppf(q, loc=0, scale=1 / lam),
         )
 
+    def test_exponential_wrong_arguments(self):
+        m = pm.Model()
```

Comment:
You don't need to use a PyMC model. Just create the variable with `pm.Exponential.dist`

Reply:
Means `m=pm.Exponential.dist` ?

Reply:
You don't need `m` or a `Model` at all.

Just do:

```python
with pytest.raises(ValueError, match=msg):
   pm.Exponential.dist(lam=0.5, scale=5)
```

Reply:
Got it. Noted.